datasets/EuRoC_MAV/README.md

```diff
@@ -0,0 +1,396 @@
+# Visualizing SLAM using Foxglove
```

Comment:
Could you please add a frontmatter section to this README.md? Then it will be nicely rendered in the main README file. You can find more details on the process here: https://github.com/foxglove/tutorials/blob/main/CONTRIBUTING. It would be good if an empty blog post is created for it. This will create a slug that you can turn into a blog post address. You can check other readmes on the process.